games-simulation/simutrans/simutrans-123.0.1-r2.ebuild

```
@@ -0,0 +1,109 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop flag-o-matic optfeature toolchain-funcs xdg
+
+MY_PV=${PV//./-}
+MY_PAK_64="simupak64-123-0.zip"
+# Required for network games, published in release announcement.
+MY_SVN_REVISION="10421"
+
+DESCRIPTION="A free Transport Tycoon clone"
+HOMEPAGE="https://www.simutrans.com/"
+SRC_URI="
+	mirror://sourceforge/simutrans/simutrans-src-${MY_PV}.zip
+	!minimal? ( mirror://sourceforge/simutrans/${MY_PAK_64} -> simutrans_${MY_PAK_64} )
+	https://tastytea.de/files/simutrans_language_pack-Base+texts-${PV}.zip
+	https://github.com/aburch/simutrans/raw/9c84822/simutrans.svg
+"
+S=${WORKDIR}
+
+# NOTE: Get the latest language pack from:
+# https://simutrans-germany.com/translator/data/tab/language_pack-Base+texts.zip
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="minimal +midi truetype upnp zstd"
+
+DEPEND="
+	app-arch/bzip2
+	media-libs/libpng:0
+	media-libs/libsdl2[sound,video]
+	midi? ( media-sound/fluidsynth[sdl] )
```

```suggestion
	midi? ( media-sound/fluidsynth:=[sdl] )
```
This should have been moved lower, below zlib.  Unconditional dependencies are grouped together first.

---

fixed, thanks! 💖